lazy_dataset/core.py

```diff
@@ -1290,6 +1295,55 @@ def cache(
             else:
                 return from_dict(dict(self))
 
+    def diskcache(
+            self,
+            cache_dir: Optional[Union[Path, str]] = None,
```

Comment:
Can you document, what the default is? A rough description is enough.

Reply:
Should be in the "Args" section